debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/AlgorithmVersion.java

```diff
@@ -71,6 +71,8 @@
 
 	public static final String MM_OBJECT_ACCESS_BARRIER_VERSION = "ALG_MM_OBJECT_ACCESS_BARRIER_VERSION";
 
+	public static final String MIXED_MODE = "MIXED_MODE";
```

Comment:
The name 'MIXED_MODE' is a little generic; it would be better if it mentioned 'references'.

Reply:
Fixed (along with the java final variable).